packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3789,10 +3788,14 @@ describe('plugin-meetings', () => {
             // that's being tested in these tests)
             meeting.webex.meetings.registered = true;
             meeting.webex.internal.device.config = {};
-            sinon.stub(MeetingUtil, 'joinMeeting').resolves({
-              id: 'fake locus from mocked join request',
-              locusUrl: 'fake locus url',
-              mediaId: 'fake media id',
+            sinon.stub(MeetingUtil, 'joinMeeting').callsFake(async (meeting) => {
+              const result = {
+                id: 'fake locus from mocked join request',
+                locusUrl: 'fake locus url',
+                mediaId: 'fake media id',
+              };
+              meeting.setLocus(result)
```

Comment:
this is getting a bit messy... maybe we could instead of stubbing `MeetingUtil.joinMeeting()` just stub `meeting.meetingRequest.joinMeeting()`?